src/daceInterface.ts

```diff
@@ -497,6 +498,12 @@ implements MessageReceiverInterface {
     public promptStartDaemon() {
         if (this.daemonBooting)
             return;
+
+        // If the optimization panel isn't open, we don't want to interact
+        // with the daemon. Don't prompt in that case.
+        if (!OptimizationPanel.getInstance().isVisible())
```

Comment:
I'd say not just the prompt, but also the whole daemon should not start until the optimization panel is visible

Reply:
The way it is right now, it actually shouldn't do that - the daemon is only called once one of the optimization panels get activated. I added a sanity check to make sure it doesn't happen regardless

Reply:
Thanks